src/Admin/Settings/Help.php

```
@@ -4,61 +4,61 @@
-						<p class="signature">
-							<a target="_blank" title="<?php echo __( 'Visit FFW Press', 'gdpr-press' ); ?>" href="https://daan.dev/wordpress-plugins/"><img class="signature-image" alt="<?php echo __( 'Visit FFW Press', 'gdpr-press' ); ?>" src="<?php echo esc_url( plugin_dir_url( GDPRESS_PLUGIN_FILE ) . 'assets/images/logo-color.png' ); ?>" /></a>
-						</p>
-					</div>
-				</div>
-			</div>
-		</div>
-		</div>
-		<?php
-	}
+    public function content() {
+        $tweet_url = sprintf( 'https://twitter.com/intent/tweet?text=I+am+using+%s+for+@WordPress!+Eliminate+external+requests+and+increase+GDPR+compliance.+Try+it+for+yourself:&via=Dan0sz&hashtags=GDPR,Privacy,WordPress&url=%s', str_replace( ' ', '+', apply_filters( 'gdpress_settings_page_title', 'GDPRess' ) ), apply_filters( 'gdpress_help_tab_plugin_url', 'https://wordpress.org/plugins/gdpr-press/' ) );
```

PHPStan (2.1.44)

[error] Function apply_filters not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(function.notFound)